lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -89,9 +91,27 @@ class FullPageScreenshot extends FRGatherer {
       screenOrientation: {angle: 0, type: 'portraitPrimary'},
     });
 
-    // TODO: elements collected earlier in gathering are likely to have been shifted by now.
-    // The lower in the page, the more likely (footer elements especially).
-    // https://github.com/GoogleChrome/lighthouse/issues/11118
+    // Now that the viewport is taller, give the page some time to fetch new resources that
+    // are now in view.
+    const networkMonitor = new NetworkMonitor(context.driver.defaultSession);
+    const waitForNetworkIdleResult = waitForNetworkIdle(session, networkMonitor, {
```

Comment:
Don't have great reasons for 1000ms or 2-idle vs 0-idle.

Reply:
you can pull `networkQuietThresholdMs` from the pass settings.  the other two are correct.

Reply:
let's do `network-critical-busy` because we don't want to wait for tracking crap

I feel that option as used for the page load isn't quite the same as this use case... besides, that would be 5s by default which is the same time as the timeout we chose :P